src/content-single/WeekendContentItem/index.js

```diff
@@ -45,70 +45,54 @@ const LiveAwareLabel = withTheme(({ isLive, title, theme }) => ({
 
 const WeekendContentItem = ({ content, loading }) => {
   const coverImageSources = get(content, 'coverImage.sources', []);
-
   return (
     <ThemeConsumer>
-      {(theme) => {
-        const overlayColor =
-          get(content, 'theme.colors.primary') ||
-          (loading ? theme.colors.lightTertiary : theme.colors.primary);
-        return (
-          <BackgroundView>
-            <StretchyView>
-              {({ Stretchy, ...scrollViewProps }) => (
-                <FlexedScrollView {...scrollViewProps}>
-                  <Header hasMedia={content.videos && content.videos.sources}>
-                    <ThemeMixin
-                      mixin={{
-                        type: (
-                          get(content, 'theme.type') || 'dark'
-                        ).toLowerCase(),
-                      }}
```

Comment:
This is also changed in the new one. I think the card is supposed to change the color of the text based on the content. So we need this, I think.